specs/eid-hermes-commands-format.md

```diff
@@ -0,0 +1,263 @@
+# eid-hermes Commands
+
+## Introduction
+
+This document defines the commands that are available for interacting with a
```

Comment:
@iomartin I would remove the work kernel in the commit log and just use "driver". Someone could write a userspace driver for Hermes and in that case the term kernel could be misleading.